project/Dependencies.scala

```diff
@@ -124,7 +126,8 @@ object Dependencies {
 
   lazy val httpTestkit = l ++= Seq(
     Test.junit, Test.junitIntf, Compile.junit % "provided",
-    Test.scalatest.value.copy(configurations = Some("provided; test"))
+    Test.scalatest.value.copy(configurations = Some("provided; test")),
+    "org.specs2" %% "specs2-core" % specs2Version.value % "provided"
```

Comment:
I've added the license info - which I believe is MIT (although I got that from maven-central so...)